api/apps/api/test/integration/typeorm-export.repository.integration.e2e-spec.ts

```diff
@@ -179,5 +194,15 @@ const getFixtures = async () => {
         });
       }
     },
+    ThenAllExportComponentsShouldBeFinished: ({
+      exportData,
+    }: {
+      exportData: Export | undefined;
+    }) => {
+      expect(exportData).toBeDefined();
+      exportData!.toSnapshot().exportPieces.map((piece) => {
+        expect(piece.finished).toEqual(true);
+      });
```

Comment:
assertions in mapping look really strange ; )

`[{finished: true}, {finished: false}].every(i => i.finished)`

Reply:
I'll fix it with a `forEach` :+1:

Reply:
@aciddaute but why `forEach`, if you can do that with `every()` ?

Reply:
Oh sorry, I didn't see that, I'll use `every()` ty Kamil 🙌